.env.example

```
@@ -1,91 +1,109 @@
 # JsonRPC Server Configuration
-RPCPROTOCOL         = 'http'
-RPCHOST             = 'jsonrpc'
-RPCPORT             = '4000'
-RPCDEBUGPORT        = '4678'      # debugpy listening port
-JSONRPC_REPLICAS    = '1'         # number of JsonRPC container replicas to run, used to scale up for production
+RPCPROTOCOL="http"
+RPCHOST="jsonrpc"
+RPCPORT="4000"
+RPCDEBUGPORT="4678"
+JSONRPC_REPLICAS="1"
+ENABLE_DEBUG_ENDPOINTS="false"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)